guides/common/modules/proc_installing-webhooks-plugin.adoc

```diff
@@ -9,11 +9,5 @@ After installing webhooks, you can configure {ProjectServer} to send webhook req
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {foreman-installer} --enable-foreman-plugin-webhooks
```

Comment:
This must remain. It was only that `{foreman-installer} --enable-foreman-cli-plugin-webhooks` is a proper replacement for `yum install tfm-rubygem-hammer_cli_foreman_webhooks`.

Reply:
Addressed :+1: